src/java_bytecode/Makefile

```diff
@@ -38,8 +38,18 @@ include ../common
 
 CLEANFILES = java_bytecode$(LIBEXT)
 
+clean : clean_library
```

Comment:
Is there a whitespace after `clean`? I'm not sure what make's semantics of this are, but I'd rather avoid it. The same seems the be true of the `clean_library` goal.